XamlStyler3.Package/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="0c572937-0d4a-48cc-aa3f-875ffa9805ba" Version="3.1" Language="en-US" Publisher="Xavalon" />
+        <Identity Id="0c572937-0d4a-48cc-aa3f-875ffa9805ba" Version="3.1.1" Language="en-US" Publisher="Xavalon" />
```

Comment:
Can you remove version bumps? We do this as we prep a release (hopefully at some point move to automated based on builds)

Reply:
should be done